### ISSUE TITLE
Class ManagerRegistry is deprecated

### ISSUE THREAD

**Issue by abelardolg** [symfony/maker-bundle#519]
PHPStorm warns about this situation:
![image](https://user-images.githubusercontent.com/22522883/70850182-c1db1480-1e87-11ea-9107-c762a034cad9.png)
Recently updated.

Best regards.

**Comment by michaljusiega** [comment]
It's related to https://github.com/doctrine/persistence/pull/71 ?

**Comment by michaljusiega** [comment]
See https://github.com/symfony/maker-bundle/pull/518

**Comment by abelardolg** [comment]
Thanks, @michaljusiega :) 

I changed from:
 use Doctrine\Common\Persistence\ManagerRegistry;
to:
use Symfony\Bridge\Doctrine\ManagerRegistry;

Thanks for your supporting! :+1:

**Comment by manzwebdesigns** [comment]
I am getting the same issue...
```
$ composer auto-scripts
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 1
!!
!!  In DefinitionErrorExceptionPass.php line 54:
!!
!!    Cannot autowire service "App\Repository\UsersRepository": argument "$registry" of method "__construct()" references
!!     class "Symfony\Bridge\Doctrine\ManagerRegistry" but no such service exists. You should maybe alias this class to t
!!    he existing "doctrine" service.
```
Should I just use the deprecated class or is there a fix pending? 😛 

Bud

**Comment by abelardolg** [comment]
I run this command:

`php bin/console make:migration`

and Symfony showed the above message.

Again the same question.

**Comment by michaljusiega** [comment]
I think you must to wait to update a 3-rd party library.

**Comment by weaverryan** [comment]
If you’re having a deprecation or error, could you please post the exact versions you’re using for MakerBunder and all libraries starting with doctrine/ ? There are a lot of subtle version things happening :)

**Comment by abelardolg** [comment]
Hi @weaverryan :    
    "symfony/maker-bundle": "^1.14",
        "doctrine/doctrine-bundle": "^2.0",

**Comment by manzwebdesigns** [comment]
Hi, here are mine...

required:
```
        "doctrine/event-manager": "~1.0",
        "doctrine/persistence": "^1.3",
        "doctrine/doctrine-bundle": "^2.0",
```
dev:
```
        "doctrine/doctrine-fixtures-bundle": "^3.2",
        "symfony/maker-bundle": "^1.14",
```

**Comment by cybernet** [comment]
`use Doctrine\Persistence\ManagerRegistry;` should be used :)

**Comment by weaverryan** [comment]
Fully fixed in #570 :). Thanks!
